### DISCUSSION TITLE
Ability to turn off the status prompt bar totally (Also, adjustment to help prompt)

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#93] [Ideas]
I typed help prompt to try to figure out how to hide the health bar, I did not see the option or it was not clear if it existed.

**Comment by Volte6** [comment]
It never occured to me that players wouldn't want a status bar at all... what would show instead? Just an empty line?

**Reply by ArmoredArtist** [reply]
It looks much cleaner and makes the screen appear less cluttered. It would be nice to see during combat only. (Or as some kind of web client graphic in the corner)

**Reply by ArmoredArtist** [reply]
This is of course user preference so it should be togglable off or on. I think a lot of people would like to see it all the time if they are used to it.

**Reply by Volte6** [reply]
That shows a `>` - did you try setting your prompt to just have that?

**Reply by ArmoredArtist** [reply]
I did not, it was not clear that I could to me. But I may have not understood the help file.

**Comment by ArmoredArtist** [comment]
I turns out there was an option to set this to "none" and I did not see it. This solved it for me and is my mistake.

**Comment by windu-ant** [comment]
Piggy backing on this thread to request a "new line" token for the prompt. Something that sends "\n". I'd prefer if each thing appeared below the prompt, instead of your commands appearing at the end of the prompt in cases like these.

![image](https://github.com/user-attachments/assets/516e3357-4a7f-4774-b6c6-3c981f32a4ce)

**Reply by Volte6** [reply]
This will be in the next update

**Comment by Volte6** [comment]
Addressed in https://github.com/Volte6/GoMud/pull/101

Closing.
